src/verinfast/agent.py

```diff
@@ -324,7 +324,8 @@ def parseRepo(self, path: str, repo_name: str):
                 "metadata": {
                     "env": machine,
                     "real_size": real_size,
-                    "uname": system
```

Comment:
This will cause an error when run on a local repo. Make sure to check if branch is actually defined.